### PR TITLE
feature/add_http_proxy add cli --proxy variable

### PR DESCRIPTION
**Pull request by oldbig**
### The subscription:
```json
{ 
"endpoint": "https://fcm.googleapis.com/fcm/send/d61c5u920dw:APA91bEmnw8utjDYCqSRplFMVCzQMg9e5XxpYajvh37mv2QIlISdasBFLbFca9ZZ4Uqcya0ck-SP84YJUEnWsVr3mwYfaDB7vGtsDQuEpfDdcIqOX_wrCRkBW2NDWRZ9qUz9hSgtI3sY", 
  "expirationTime": null, 
  "keys": { 
    "p256dh": "BL7ELU24fJTAlH5Kyl8N6BDCac8u8li_U5PIwG963MOvdYs9s7LSzj8x_7v7RFdLZ9Eap50PiiyF5K0TDAis7t0", 
    "auth": "juarI8x__VnHvsOgfeAPHg" 
  } 
}
```

### The command:
```bash
web-push send-notification  \
--endpoint=https://fcm.googleapis.com/fcm/send/d61c5u920dw:APA91bEmnw8utjDYCqSRplFMVCzQMg9e5XxpYajvh37mv2QIlISdasBFLbFca9ZZ4Uqcya0ck-SP84YJUEnWsVr3mwYfaDB7vGtsDQuEpfDdcIqOX_wrCRkBW2NDWRZ9qUz9hSgtI3sY \
--key=BL7ELU24fJTAlH5Kyl8N6BDCac8u8li_U5PIwG963MOvdYs9s7LSzj8x_7v7RFdLZ9Eap50PiiyF5K0TDAis7t0 \
--auth=juarI8x__VnHvsOgfeAPHg \
--vapid-subject=mailto:example@qq.com \
--vapid-pubkey=BGtkbcjrO12YMoDuq2sCQeHlu47uPx3SHTgFKZFYiBW8Qr0D9vgyZSZPdw6_4ZFEI9Snk1VEAj2qTYI1I1YxBXE \
--vapid-pvtkey=I0_d0vnesxbBSUmlDdOKibGo6vEXRO-Vu88QlSlm5j0 \
--payload=Hello \
--proxy=http://127.0.0.1:8889
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "web-push",
-  "version": "3.4.5",
+  "version": "3.4.6",
```
Comment (marco-c) [inline package.json:3]: This should be done separately, can you revert it from the PR?

File: package.json [hunk 2]
```diff
@@ -53,4 +53,4 @@
   "engines": {
     "node": ">= 6"
   }
-}
+}
```
Comment (marco-c) [inline package.json:56]: Revert this change too

File: src/cli.js [hunk 3]
```diff
@@ -49,9 +50,9 @@ const generateVapidKeys = returnJson => {
   } else {
     const outputLine = '\n=======================================\n';
     outputText = outputLine + '\n'
-    + 'Public Key:\n' + vapidKeys.publicKey + '\n\n'
-    + 'Private Key:\n' + vapidKeys.privateKey + '\n'
-    + outputLine;
+      + 'Public Key:\n' + vapidKeys.publicKey + '\n\n'
+      + 'Private Key:\n' + vapidKeys.privateKey + '\n'
+      + outputLine;
```
Comment (marco-c) [inline src/cli.js:55]: Revert these whitespace changes too

**Review by marco-c — CHANGES_REQUESTED** [review]
Just a few minor things to fix.

Could you also update the "Command Line" section in the README with the new argument?

File: package.json (hunk 1 above)
Comment (oldbig) [reply package.json]: Ok
